segregation/local/local_indexes.py

```diff
@@ -12,6 +12,10 @@
 
 from segregation.spatial import Relative_Centralization
 
+# suppress numpy divide by zero warnings because it occurs a lot during the
+# calculation of many indices
+np.seterr(divide='ignore', invalid='ignore')
+
 
 __all__ = [
     'Multi_Location_Quocient',
```

Comment:
this should be "Quotient"

Reply:
Ow, sure, will fix this!

Reply:
> this should be "Quotient"

I think with the last commit, this is addressed. BTW, in Brazilian Portuguese, we use "quociente", that's why I got confused! hahaha